qiskit/synthesis/clifford/clifford_decompose_greedy.py

```diff
@@ -60,60 +60,69 @@ def synth_clifford_greedy(clifford: Clifford) -> QuantumCircuit:
     num_qubits = clifford.num_qubits
     circ = QuantumCircuit(num_qubits, name=str(clifford))
     qubit_list = list(range(num_qubits))
-    clifford_cpy = clifford.copy()
+    clifford_current = clifford.copy()
 
     # Reducing the original Clifford to identity
     # via symplectic Gaussian elimination
     while len(qubit_list) > 0:
-        # Calculate the adjoint of clifford_cpy without the phase
-        clifford_adj = clifford_cpy.copy()
-        tmp = clifford_adj.destab_x.copy()
-        clifford_adj.destab_x = clifford_adj.stab_z.T
-        clifford_adj.destab_z = clifford_adj.destab_z.T
-        clifford_adj.stab_x = clifford_adj.stab_x.T
-        clifford_adj.stab_z = tmp.T
 
+        # Calculate the adjoint of clifford_current without the phase
```

Comment:
is this comment still correct? do you calculate the adjoint or the evolved cliffords directly?